logoapp/js/dot_data.js

```diff
@@ -428,51 +427,55 @@ DotAppFramework.prototype.onClickUploadButton = function() {
 	var blob = new Blob([bin.buffer], {type:'image/gif'});
 
 	var b64 = window.btoa(encoder.stream().getData());
-	var url2 = 'data:image/gif;base64,'+b64;
+	//画像ファイルここまで
+	//データここから
+	clearInterval(this.completeLoopInterval	);
+	var uploadData = new Array(8);
+	for(var i=0; i<uploadData.length; i++) {

```

Comment:
__[jshint]__
*  'i' is already defined.


<!-- Comment by [SideCI](https://sideci.com) -->